stable/concourse/README.md

```diff
@@ -81,6 +81,8 @@ The following table lists the configurable parameters of the Concourse chart and
 | `web.ingress.enabled` | Enable Concourse Web Ingress | `false` |
 | `web.ingress.hosts` | Concourse Web Ingress Hostnames | `[]` |
 | `web.ingress.tls` | Concourse Web Ingress TLS configuration | `[]` |
+| `web.prometheus.enabled` | Enable Prometheus metrics endpoint | `false` |
```

Comment:
Hey, the `prometheus.enabled` thing under `web` is actually under `concourse.web.prometheus.enabled`, right? From the set of variables that are more directly mapped to `concourse web` commands (which we end up not documenting in the README.md)

Reply:
@cirocosta  I have seen these variables are not documented.
I think we might improve my pull-request and document these variables.

Reply:
Yeah, not having those indeed break the pattern that is established regarding documenting all the values under `values.yaml` in `README.md`. 

I'd say we can have a separate PR for that though - the change would involve **a bunch** of additions to the README.md. Wdyt?

Reply:
Definitely. I think this should be a separate pull-request.
I will be happy to help with that.

Reply:
👍 I just opened https://github.com/helm/charts/issues/11300 to keep track of it